packages/playground/package.json

```diff
@@ -84,40 +84,38 @@
     "@material-ui/icons": "^4.11.3",
     "@mui/icons-material": "^5.8.4",
     "atob": "^2.1.2",
+    "ajv": "^8.11.0",
```

Comment:
v8? Do we even need this?

Reply:
We do... see this remark in the commit description:?
```
Explicitly added ajv8 since webpack-dev-server uses it and the validator-ajv6 has an earlier version
```